packages/web-runtime/src/pages/account.vue

```diff
@@ -20,39 +32,35 @@
       <h2 v-translate class="oc-text-bold oc-text-initial oc-mb">Account Information</h2>
 
       <dl class="account-page-info oc-flex oc-flex-wrap">
-        <div class="oc-mb oc-width-1-2@s">
-          <dt v-translate class="account-page-info-username oc-text-normal oc-text-muted">
```

Comment:
selectors are moved one level up (not only to the title of the account info" but wrapping whole div with `dt` and `dd` inside)